frontend/app/(tabs)/index.tsx

```diff
@@ -1,6 +1,47 @@
-import { Text, View } from '@/components/Themed';
+/*  Homepage layout
+ *   Includes a background image of a recipe image
+ *   Includes a side bar with an Avatar, like button, comment
+ *   button, and a share button.
+ */
+
 import { Link } from 'expo-router';
+import { TouchableHighlight, ImageBackground } from 'react-native';
+import { Button, YStack, Image, styled, Avatar, Text, View } from 'tamagui';
 
 export default function Index() {
-  return <View></View>;
+  const backgroundImage = { uri: 'https://picsum.photos/1080/1920' };
+  const StyledBG = styled(ImageBackground, { flex: 1 });
```

Comment:
The reason why it wouldn't work was because you used `ImageBackground` from `react-native` instead of `tamagui`. There is a image or something similar but that can be fixed in a later PR.